engines/sci/graphics/palette32.h

```diff
@@ -90,22 +118,122 @@ namespace Sci {
 		inline PalCycler *getCycler(uint16 fromColor);
 
 		/**
-		 * The fade table records the expected intensity level of each pixel in the palette that will be displayed on
-		 * the next frame.
+		 * The fade table records the expected intensity level of each pixel
+		 * in the palette that will be displayed on the next frame.
 		 */
 		byte _fadeTable[256];
+
+		/**
+		 * TODO: Documentation
+		 */
```

Comment:
This is documented in loadClut() and matchClutColor().

loadClut() will load a color lookup table from a clu file and set the palette found in the file. This is to be used with Phantasmagoria 2.
matchClutColor() matches a color in RGB565 format to a palette index based on the loaded CLUT